data/starts.txt

```diff
@@ -59,18 +60,19 @@ start "hai space after FW and HR"
 	on display
 		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
 		thumbnail ""
-		date "middle of 3020"
+		date "3020"
+		planet "Slightly too warm"
 		credits "over a million"
 	to reveal
 		has "main plot completed"
 		has "Ask the Hai about the Unfettered"
 	on reveal
-		name "Hai Start"
+		name "Undiscovered Hai Start"
 		description "There could be interesting possibilities up north, once all crises are resolved."
-		description "You could have started a different way, went a different path."
-		system "in Hai space"
-		planet "human population"
-		date "Jun 3020"
+		description "	You could have started a different way, went a different path."
```

Comment:
Are you sure about that tab?

Reply:
the existing start has this as well between lines?

Reply:
Oops. I thought it was placed after the deleted lines, where it would have been the first description line. My bad.